recipes/whisper-cpp/all/conanfile.py

```diff
@@ -114,13 +150,19 @@ def generate(self):
         if self.options.no_f16c:
             tc.variables["WHISPER_NO_F16C"] = True
 
+        if self.options.get_safe("with_openvino"):
+            tc.variables["WHISPER_OPENVINO"] = True
+            tc.variables["CMAKE_CXX_STANDARD"] = 11
```

Comment:
```suggestion
            tc.variables["CMAKE_CXX_STANDARD"] = self.settings.get_safe("compiler.cppstd", 11)
```

We have few other recipes using similar approach that should fit this case too:
- Use the cppstd in case defined, otherwise, C++11. So you still enforces a c++ standard, but keep cppstd as priority when available.

Reply:
Hi @uilianries 
Thank you for your suggestion. 

I suppose this line will be completely dropped once we resolve issue with C++ standard in OpenVINO recipe. Do you think it's safe to merge https://github.com/conan-io/conan-center-index/pull/20785 to OpenVINO recipe?

Based on what I've written above, it's safe to add `target_compile_features`, because they operate in the same manner as `self.settings.get_safe("compiler.cppstd", 11)`, but on cmake level. And with my proposal this information is stored in main recipe, but not in dependent ones (e.i. we don't have to fix every users of OpenVINO recipe)

Reply:
@ilya-lavrenov I don't see it as OpenVino responsibility, because Whisper.cpp is using C++11 features (e.g. https://github.com/ggerganov/whisper.cpp/blob/v1.4.2/whisper.cpp#L705) but not defined directly. Thus, I still prefer keep only this change here and closing both #20785 and #20782

Reply:
We manage the required c++ standard only by validate(), that's why we don't export via cmake modules. For Conan 1.x is true we don't use cppstd by default, because is a feature introduced after 1.0, but still highly recommended. The result is that we need to set the required c++ standard in test package too, but it's something related to legacy, that's why we don't care so much. This is not a problem for most of cases, because usually projects set the required C++ standard in their cmakefiles.

In the future, only Conan 2.x will be running in ConanCenterIndex and cppstd will be present by default, so I would not invest so much on this now, keeping this change only in Whisper.